deeplake/core/index_maintenance.py

```diff
@@ -17,7 +17,7 @@ class INDEX_OP_TYPE(Enum):
     INCREMENTAL_INDEX = 3
 
 
-def is_embedding_tensor(tensor):
+def validate_embedding_tensor(tensor):
```

Comment:
I think here the validation should be without name-checking(just htype check should be enough). but I am not sure

Reply:
This is from long time. Looks like best to only check for htype. Dont know about any case where htype is not set to 'embedding' but tensor.meta.name and tensor.key are set to 'embedding' Removing them.